platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/components/state/StateManagementComponentTests.java

```diff
@@ -393,6 +396,36 @@ void testIssConsumer() {
         component.stop();
     }
 
+    @Test
+    @DisplayName("Test that the state is saved to disk when it is received via reconnect")
+    void testReconnectStateSaved() throws InterruptedException {
+        final Random random = RandomUtils.getRandomPrintSeed();
+        final AddressBook addressBook = new RandomAddressBookGenerator(random)
+                .setSize(NUM_NODES)
+                .setWeightDistributionStrategy(WeightDistributionStrategy.BALANCED)
+                .setSequentialIds(false)
```

Comment:
Shouldn't this be `true`, since you're assuming contiguous IDs when collecting `majorityWeightNodes` below?

Reply:
good catch, made it true